tests/test_fingerprints.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for fingerprint generation."""
+
+import pytest
+
+from baybe._optional.info import CHEM_INSTALLED
+from baybe.parameters.enum import SubstanceEncoding
+from baybe.utils.chemistry import smiles_to_fingerprint_features
+
+test_cases: list[tuple[SubstanceEncoding, dict, dict]] = [
+    (enc, {}, {})
+    for enc in SubstanceEncoding
+    if enc
+    not in {  # Ignore deprecated encodings
+        SubstanceEncoding.MORGAN_FP,
+        SubstanceEncoding.RDKIT,
+    }
+]
+
+ECFP = SubstanceEncoding.ECFP
+
+
+@pytest.mark.skipif(
+    not CHEM_INSTALLED, reason="Optional chem dependency not installed."
+)
+@pytest.mark.parametrize(
```

Comment:
Do I understand this correctly that `kw_fp` and `kw_conf` are just empty dicts for all tests but the manually added ones? Is this intentional? Note that I have no idea about fingerprints, so this might be a stupid question :D

Reply:
these are for optinal configuration of the fingeprints

So for the test we iterate over all fingeprrints (with the dicsts emtpy) + add a few cases with non-empty dicts to test the configurability

Reply:
I think this is fine and will close the issue now in order to move forward. @AVHopp: feel free to reopen if necessary